common/docker-entrypoint.sh

```diff
@@ -1,25 +1,78 @@
 #!/bin/bash
-set -eu
+set -e
+
+# DEBUG flag to enable debug output. To turn on debug set DEBUG=1 (e.g. docker compose up -d --build --env DEBUG=1)
+TEMP_DEBUG="${DEBUG:-0}"
+if [[ "$TEMP_DEBUG" == "1" || "${TEMP_DEBUG}" == "true" || "${TEMP_DEBUG}" -eq 1 ]]; then
```

Comment:
decide between one of the options. Suggestion:

```suggestion
if [[ "${TEMP_DEBUG}" -eq 1 ]]; then
```

Reply:
Also, why to we need the TEMP_DEBUG variable on the first place?

Reply:
Isn't it better to check if DEBUG = 0 or DEBUG = 1, and fail if that's not the case?

This way we "teach" users, instead of compiling with them not reading the readme.

Reply:
Or are you meaning that even if we set DEBUG = 1, docker might interpret it on other ways?

Reply:
> Or are you meaning that even if we set DEBUG = 1, docker might interpret it on other ways?

it depends on _how_ the flag is set, if memory serves correctly. If the flag is set via cli it is string but it could be a number if set via docker arg.

Let me verify these needs

Reply:
Hey, let's rely on "true" and "false" for this ENV, to make it consistent with the other ones